app/main/auto-updater/index.js

```diff
@@ -7,88 +7,99 @@
 import { app, autoUpdater, dialog } from 'electron';
 import moment from 'moment';
 import B from 'bluebird';
+import _ from 'lodash';
 import { getUpdate } from './update-checker';
 import { getFeedUrl } from './config';
+
+const isDev = process.env.NODE_ENV === 'development';
   
-autoUpdater.setFeedURL(getFeedUrl(app.getVersion()));
+let checkNewUpdates = _.noop;
 
-/** 
- * Check for new updates
- */
-export async function checkNewUpdates (fromMenu) {
-  // autoupdate.checkForUpdates always downloads updates immediately 
-  // This method (getUpdate) let's us take a peek to see if there is an update 
-  // available before calling .checkForUpdates
-  const update = await getUpdate(app.getVersion());
-  if (update) {
-    let {name, notes, pubDate:pubDate} = update;
-    pubDate = moment(pubDate).format('MMM Do YYYY, h:mma');
+if (!isDev) {
+
+  autoUpdater.setFeedURL(getFeedUrl(app.getVersion()));
+
+  /** 
+   * Check for new updates
+   */
+  checkNewUpdates = async function (fromMenu) {
+    // autoupdate.checkForUpdates always downloads updates immediately 
+    // This method (getUpdate) let's us take a peek to see if there is an update 
+    // available before calling .checkForUpdates
+    const update = await getUpdate(app.getVersion());
+    if (update) {
+      let {name, notes, pubDate:pubDate} = update;
```

Comment:
`pubDate:pubDate` shouldn't be necessary

Reply:
Good catch, that should be `pub_date:pubDate`